keyboards/levinson/keymaps/atreus/rules.mk

```diff
@@ -0,0 +1,5 @@
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
+
+USE_I2C=yes
```

Comment:
This doesn't actually do anything, and can be removed.

```suggestion
```

In fact, you'd need `#define USE_I2C` in your config.h.